quodlibet/po/fr.po

```diff
@@ -4565,35 +4571,36 @@ msgstr "Éditer les étiquettes"
 
 #: ../quodlibet/qltk/edittags.py:477
 msgid "Show _programmatic tags"
-msgstr "Montrer les commentaires de _programmation"
+msgstr "Afficher les étiquettes _programmables"
```

Comment:
Not sure about "programmable".. "programmatique" exists, see http://www.larousse.fr/dictionnaires/francais/programmatique/64206